src/pages/Home.jsx

```diff
@@ -148,45 +154,30 @@ const Home = () => {
     },
   ];
 
-  const loadListings = async () => {
-
+  const handleFilterChanged = async (params) => {
     setLoading(true)
-    const auctionListings = await getAuctionListings(
-      contracts.MarketContract,
-      contracts.ZoombiesContract,
-      filters,
-      sortBy
-    );
-    setListings(auctionListings.map((listing) => ({
-      ...listing,
-      id: listing._id
-    })));
-    setLoading(false)
-  };
-
-
-
-  const handleRowClick = ({row}) => {
-    history.push(`/listing/${row.itemNumber}`);
-  };
-
-  const handleFilterChanged = (params) => {
     setFilters({ ...filters, ...params })
+    remove()
   }
 
   const handleSortByChanged = (attribute) => {
-    setSortBy({ ...sortBy, ...attribute })
+    // setSortBy({ ...sortBy, ...attribute })
+    setLoading(true)
+    setFilters({ ...filters, ...{
+      sortField: attribute.field
+    }})
+    remove()
```

Comment:
OOC what does this remove do?

Reply:
remove() - this is for removing caches (when we change the filter conditions and sort fields)
or it doesn't refresh the fetched auctions.